README.md

```diff
@@ -12,43 +12,43 @@ This repository contains examples of various algorithms written on different pro
-| [Maximum Subarray Problem](https://en.wikipedia.org/wiki/Maximum_subarray_problem)              |                                       |                                       |                                       | [:octocat:](/maximum_subarray/Python) |
-| [Knapsack Problem](https://en.wikipedia.org/wiki/Knapsack_problem)                              |                                       |                                       |                                       | [:octocat:](knapsack_problem/Python)  |
-| [Selecton Sort](https://en.wikipedia.org/wiki/Selection_sort)                                   | [:octocat:](selection_sort/C)         | [:octocat:](selection_sort/Cpp)       |                                       |                                       |
+| Algorithm                                                                          |               C               |               CPP               |              Java               |                Python                 |
+| :--------------------------------------------------------------------------------- | :---------------------------: | :-----------------------------: | :-----------------------------: | :-----------------------------------: |
+| [Euclidean GCD](https://en.wikipedia.org/wiki/Euclidean_algorithm)                 | [:octocat:](euclidean_gcd/C)  |                                 | [:octocat:](euclidean_gcd/Java) |   [:octocat:](euclidean_gcd/Python)   |
+| [Merge Sort](https://en.wikipedia.org/wiki/Merge_sort)                             |   [:octocat:](merge_sort/C)   |   [:octocat:](merge_sort/Cpp)   |  [:octocat:](merge_sort/Java)   |    [:octocat:](merge_sort/Python)     |
+| [QuickSort](https://en.wikipedia.org/wiki/Quicksort)                               |   [:octocat:](quicksort/C)    |                                 |   [:octocat:](quicksort/Java)   |     [:octocat:](quicksort/Python)     |
+| [Insertion Sort](https://en.wikipedia.org/wiki/Insertion_sort)                     | [:octocat:](insertion_sort/C) | [:octocat:](insertion_sort/Cpp) |                                 |  [:octocat:](insertion_sort/Python)   |
+| [Counting Sort](https://en.wikipedia.org/wiki/Counting_sort)                       | [:octocat:](counting_sort/C)  | [:octocat:](counting_sort/Cpp)  |                                 |   [:octocat:](counting_sort/Python)   |
+| [Radix Sort](https://en.wikipedia.org/wiki/Radix_sort)                             |                               |   [:octocat:](radix_sort/Cpp)   |                                 |    [:octocat:](radix_sort/Python)     |
+| [Binary Search](https://en.wikipedia.org/wiki/Binary_search_algorithm)                          | [:octocat:](binary_search/C) | [:octocat:](binary_search/Cpp)        | [:octocat:](binary_search/C)          | [:octocat:](binary_search/Python)     |
```

Comment:
You have listed you file at 2 places in this line, one in `C` column and other in `Java` column. I hope you understand what you are doing here